fe/fe-core/src/main/java/com/starrocks/qe/scheduler/Deployer.java

```diff
@@ -104,12 +108,19 @@ public void deployFragments(List<ExecutionFragment> concurrentFragments, boolean
         }
 
         for (List<FragmentInstanceExecState> executions : twoStageExecutionsToDeploy) {
-            try (Timer ignored = Tracers.watchScope(Tracers.Module.SCHEDULER, "DeployStageByStageTime")) {
-                executions.forEach(FragmentInstanceExecState::deployAsync);
-            }
-
-            try (Timer ignored = Tracers.watchScope(Tracers.Module.SCHEDULER, "DeployWaitTime")) {
-                waitForDeploymentCompletion(executions);
+            // Fragment Instance carrying runtime filter params for runtime filter coordinator
+            // must be delivered at first.
+            Map<Boolean, List<FragmentInstanceExecState>> partitionedBatches = executions.stream()
```

Comment:
`batches` makes `twoStageExecutionsToDeploy` be not a truth anymore. 

Maybe move this logic to `createFragmentInstanceExecStates`, which generates `twoStageExecutionsToDeploy` for now, and make it generate three stage executions? Then, this `for` loop still just do nothing but deploy the requests.
- stage 1, the request with RF coordinator.
- stage 2, the first request to a host, which need send descTable.
- stage 3, the non-first requests to a host, which needn't send descTable.